internal/kubernetes/operator/features/crds.go

```diff
@@ -83,6 +84,19 @@ var (
 			resource{ResourceAtlasStreamInstance, NopPatcher()},
 			resource{ResourceAtlasStreamConnection, NopPatcher()},
 		},
+		"2.4.0": {
```

Comment:
You should remove the 2.1.0 entry. Unless something has changed we only support the 3 latest (major) versions of AKO

Reply:
Done